tests/integration/orcid/fixtures/test_orcid_domain_models_TestOrcidPusherDuplicatedIdentifier_clashing.json

```diff
@@ -31,5 +31,6 @@
   "collaborations": [],
   "document_type": ["article"],
   "preprint_date": "2017-05",
-  "control_number": 666
+  "control_number": 666,
+  "deleted": true
```

Comment:
I'm not sure this is correct, as you're not pushing that record if it's deleted from the start. If you look at the cassette, you'll see that you're just pushing the other record and that's it, so you're not testing this feature.